packages/c/srv/src/srv.c

```diff
@@ -190,17 +190,30 @@ int run_srv_daemon_side_multi(srv_params_t *params) {
                      "run_srv_daemon_side_multi\n Control socket received %s request - \n creating new socketToSocket "
                      "connection\n",
                      messagetype);
-        // start socket_to_socket connection
-        res = create_encrypter_and_decrypter(new_session_aes_key_string, new_session_aes_iv_string,
-                                             new_socket_encrypter, new_socket_decrypter);
+
+        bool no_encrypt =
+            strcmp(new_session_aes_key_string, "no") == 0 && strcmp("new_session_aes_iv_string", "encrypt") == 0;
+        if (no_encrypt) {
+          atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_WARN,
+                       "Socket connector requested no encryption!\n\tOnly disable encryption if you know what you "
+                       "are doing!\n");
+        }
```

Comment:
no_encrypt is currently always false, I put this code in the wrong place when I initially started this branch. This code is functionally equivalent to what it was before and I will resolve it in a later iteration